ports/stm32/mboot/main.c

```diff
@@ -441,6 +441,8 @@ static int usrbtn_state(void) {
 /******************************************************************************/
 // FLASH
 
+#define APPLICATION_FLASH_LENGTH (FLASH_END + 1 - APPLICATION_ADDR)
```

Comment:
for now, to get this building (and working) on WB55 I think put the following above this line:
```c
#if defined(STM32WB)
#define FLASH_END FLASH_END_ADDR
#endif 
```